src/config.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 
 export interface Config {
+  keycloak: any;
```

Comment:
I think there is a Keycloak configuration type in keycloak-js

Reply:
skal prøve, takk

Reply:
initOptions er der, men ikke constructor options. kc selv bruker union type "string|{}"